docs/ops/convolution/Convolution_1.md

```diff
@@ -1,35 +1,35 @@
-## Convolution<a name="Convolution"></a> {#openvino_docs_ops_convolution_Convolution_1}
+## Convolution <a name="Convolution"></a> {#openvino_docs_ops_convolution_Convolution_1}
 
 **Versioned name**: *Convolution-1*
 
 **Category**: Convolution
 
-**Short description**: [Reference](http://caffe.berkeleyvision.org/tutorial/layers/convolution.html)
+**Short description**: Computes 1D, 2D or 3D convolution (cross-correlation to be precise) of input and kernel tensors.
 
-**Detailed description**: [Reference](http://cs231n.github.io/convolutional-networks/#conv)
+**Detailed description**: [Convolutional Neural Networks](http://cs231n.github.io/convolutional-networks/#conv), [Convolution operation](https://medium.com/apache-mxnet/convolutions-explained-with-ms-excel-465d6649831c).  
```

Comment:
Please, add a one paragraph description how in general the convolution works.

Reply:
All comments regarding spec done in this [commit](https://github.com/openvinotoolkit/openvino/pull/3922/commits/8293981e3cfd233fe38fcd2106d71798f4621ea4). Describing Convolution in one sentence was the most difficult part of this PR! So, if you have better description, please share it with me :)